BoxContentSDK/BoxContentSDK/OAuth2/BOXOAuth2Session.h

```diff
@@ -80,6 +80,11 @@ extern NSString *const BOXOAuth2UserIDKey;
  */
 @property (nonatomic, readwrite, weak) BOXAPIQueueManager *queueManager;
 
+/**
+ * The randomly generated nonce used to prevent spoofing attack during login
+ */
+@property (nonatomic, readwrite, strong) NSString *nonce;
```

Comment:
Does this need to be public? I don't see it referred outside of the OAuth2Session class' implementation.